### PR TITLE
Host status selector

### PR DESCRIPTION
**Pull request by jtomasek**
This pull request adds BaremetalHostModel definition to models, introduces basic getHostStatus selector to be used with hosts filtering and determining host status.

### CODE REVIEW

File: src/components/HostStatus/getHostStatus.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import * as _ from 'lodash';
+
+export const getHostStatus = host => _.get(host, ['metadata', 'labels', 'metalkube.org/operational-status']);
```
Comment (suomiy) [inline src/components/HostStatus/getHostStatus.js]: this should probably be in `utils/selectors` as it is not a component. Or make a file  `utils/hostSelectors`,  if you think there will be more of them

Comment (jtomasek) [reply src/components/HostStatus/getHostStatus.js]: I have taken an inspiration for this in VmStatus/getVmStatus.js selector which is colocated to the component. I expect this selector to grow in a similar manner and a statu component to be added. But yeah, I am happy to move it under utils as you're saying.

Comment (suomiy) [reply src/components/HostStatus/getHostStatus.js]: `getVmStatus.js` does a bit more than just data selection and it belongs to `VmStatus` component. Nevertheless some of the functionality from `getVmStatus.js` could be refactored to some selectors file.

Comment (jtomasek) [reply src/components/HostStatus/getHostStatus.js]: Done

File: src/index.js [hunk 2]
```diff
@@ -20,5 +20,6 @@ export * from './components/ConfigurationSummary';
 // helpers
 export * from './constants';
 export * from './utils';
+export * from './models';
```
Comment (rawagner) [inline src/index.js]: Why do you want to export models ? We usually have all models in `web-ui` and `web-ui-components` contains only copies of necessary models from `web-ui`

Comment (jtomasek) [reply src/index.js]: The TODO comment in `kubevirt/modes/vm.ts` [1] suggest reusing the models from web-ui-components. Exporting models here allows doing that. If I am testing it correctly, then I just need to re-export the model in `metalkube/models/host.ts` which I find better than maintaining 2 copies in 2 repositories.

Comment (rawagner) [reply src/index.js]: makes sense :)

**Pull request merged**